### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: Jaws and NVDA are not annoucing the message when list is empty in Run on web app for Containers Dialog

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1499]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Navigate to and select created Project ->Azure->Run on web app for containers.
4. "Run on web app for Containers" dialog will open.
5. Tab to and select Use Existing web app or create new web app radio button radio button is selected
6. Navigate to  "List" which is present under Before launch:Build Artifacts,Activate Tool window then observe the NVDA and JAWS Behavior

**Expected Result:**
NVDA and Jaws should announce "There are no tasks to run before launch"

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as list
**Jaws Behavior:**
List view
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![by using shortcut key for add nvda annoucning some extra information in run on web app for containers window](https://user-images.githubusercontent.com/22956130/36137181-fb59c5fe-10b9-11e8-91bf-99f1ae720d0d.jpg)
![run on web app for containers-alt-insert](https://user-images.githubusercontent.com/22956130/36137230-3e37768c-10ba-11e8-99b1-c2fbe5287af3.jpg)

**Comment by Eskibear** [comment]
The image seems not to match the issue.  

And the content is provided by IntelliJ run/debug configuration, out of the toolkit's control.

**Comment by Eskibear** [comment]
and it's duplicate with #1501

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749943
